finalPrep/annotations.js

```diff
@@ -15,28 +21,37 @@ function buildABear(name, age, fur, clothes, specialPower) {
     isCuddly: true,
   };
 
+// returns variable builtBear and includes each key-value pair
   return builtBear
 }
 
+// defines 2 bears with different parameters.
```

Comment:
For these two lines, `call the buildABear function with different arguments` would be a little more accurate